list_of_prime_numbers/main.py

```diff
@@ -1,18 +1,33 @@
 
 def _is_prime(number):
-    pass
+    if number == 2:
+        return True
+    else:
+        for i in range(2,number):
+            if (number % i) == 0:
+                return False
+    return True
 
 
 def list_of_prime_numbers(max_number):
-    pass
-
+    primeList = []
+    for i in range(2,max_number + 1):
```

Comment:
I like how you started the range at 2. It skips the unnecessary loops for 0 and 1. 
You might want to remove `print(primeList)` from your final code as it doesn't add anything to the function.
Overall, great job!